2024/calcite-react/demo/src/main.tsx

```diff
@@ -1,28 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
+import esriConfig from '@arcgis/core/config';
+
 import App from './App.tsx';
 
-// Import CSS
+// Step 1: Import CSS
 import './index.css';
 
-// JSON containing values to configure app
+// Step 3 (OPTIONAL): JSON containing values to configure app
```

Comment:
for an optional step, it shouldn't be screaming too loud 😁

```suggestion
// Step 3 (optional): JSON containing values to configure app
```

also, did you miss step 2?